controller/scheduler/main.go

```diff
@@ -169,6 +181,7 @@ func (c *context) syncCluster() {
 				ReleaseID: releaseID,
 				Type:      jobType,
 				State:     "up",
+				Meta:      jobMetaFromMetadata(job.Metadata),
```

Comment:
Given the scheduler does not change the metadata, does it need to send it here? I know `PUT` requests are supposed to put the complete resource, maybe change it to `PATCH`?